cmd/pipecd/ops.go

```diff
@@ -212,3 +217,8 @@ func ensureSQLDatabase(ctx context.Context, cfg *config.ControlPlaneSpec, logger
 	logger.Info("prepare SQL schema and indexes successfully")
 	return nil
 }
+
+func registerOpsMetrics(col ...prometheus.Collector) {
+	r := prometheus.DefaultRegisterer
```

Comment:
How about prefixing this with `pipecd_ops_`?
And update the prefix for the server to be `pipecd_server_` too.

Reply:
They can be identified by the `job` label so I thought it's unneeded. But to unify, that's better

Reply:
👍

Reply:
Okay then plz update prefix for server in this pr 🙏 https://github.com/pipe-cd/pipe/pull/2259#discussion_r670925821
Will address ops prefix here 🙆‍♀️

Reply:
Done by https://github.com/pipe-cd/pipe/pull/2258/commits/1cd410e22f1f84e1bcb25a82eb2ae9c6aad2ed7f 👌

Reply:
got it